Jenkinsfile

```diff
@@ -79,6 +80,12 @@ def ARTIFACTORY_RELEASE_REPO = "libs-release-local"
 */
 def ZOWE_LICENSE_ZIP_PATH = "/org/zowe/licenses/1.0.0/zowe_licenses_full.zip"
 
+/**
+* The locations where the pipeline will look for the License Zip
+*/
+//def ZOWE_LICENSE_ZIP_URL = "https://zowe.jfrog.io/zowe/$ARTIFACTORY_RELEASE_REPO$ZOWE_LICENSE_ZIP_PATH"
+def ZOWE_LICENSE_ZIP_URL = "https://wash.zowe.org:8443/job/Zowe%20Dependency%20Scan%20-%20Multibranch/job/staging%252Fv$ZOWE_VERSION_NUMBER/lastSuccessfulBuild/artifact/zowe_licenses_full.zip"

```

Comment:
I see this as something that could be very fragile in the sense that anything in here could change.
It'd be nice to have some error handling around it. or at least hat it parameterized.
No need to change it now though.